js/views/storyeditview/passageDSEditor.js

```diff
@@ -14,6 +14,10 @@ StoryEditView.PassageDSEditor = Backbone.View.extend({
     this.$('.passageName').val(this.model.get('name'));
     var text = this.model.get('text');
     this.$('.passageText').val((text == Passage.prototype.defaults.text) ? '' : text);
+    this.$('#edit-ds-oip').val(this.model.get('optinpath'));
+    this.$('#edit-group-success-path').val(this.model.get('groupSuccessPath'));
+    this.$('#edit-indiv-success-path').val(this.model.get('indivSuccessPath '));
```

Comment:
There's an extra space in the `'indivSuccessPath '` string that makes model data not get loaded into the modal.